dashboard/settings/base.py

```diff
@@ -151,6 +151,12 @@
 
 FLOAT_API_TOKEN = os.environ.get('FLOAT_API_TOKEN')
 
+ADMIN_HEADINGS = {
```

Comment:
This doesn't look to be used. I think it is left over from my initial start on this